src/runtime/auto-prompt-manager.ts

```diff
@@ -190,7 +199,10 @@ export class AutoPromptManager {
       this.autoPromptType_ = this.getPromptTypeToDisplay_(
         params.autoPromptType
       );
-      this.isClosable_ = params.isClosable ?? !this.isSubscription_();
+      this.isClosable_ = this.actionOrchestrationExperiment_
```

Comment:
Should we move up `setArticleExperimentFlags_` invocation so that the flag value is set at this point?